src/configs/substrateProviders.ts

```diff
@@ -37,40 +35,35 @@ interface ChainConfigs {
   [chain: string]: ChainConfig;
 }
 
-// create a custom hasher (512 bits, combo of blake2 and keccak)
-function hasherH512(data: any) {
-  return u8aConcat(blake2AsU8a(data), keccakAsU8a(data));
-}
+const getChainNames = () => {
+  const chainVarName = 'REACT_APP_CHAIN_';
+  return [checkEnvVariable(`${chainVarName}1`), checkEnvVariable(`${chainVarName}2`)];
```

Comment:
```suggestion
  return [
    checkEnvVariable('REACT_APP_CHAIN_1'),
    checkEnvVariable('REACT_APP_CHAIN_2')
  ];
```
This is overkill :) Having variables names explicit makes it much easier to look for them.